pkg/services/featuremgmt/registry.go

```diff
@@ -208,9 +208,9 @@ var (
 			FrontendOnly: true,
 		},
 		{
-			Name:        "prometheusStreamingJSONParser",
-			Description: "Enable streaming JSON parser for Prometheus datasource",
-			State:       FeatureStateBeta,
+			Name:        "prometheusBufferedClient",
+			Description: "Use buffered (old) client as default instead of streaming JSON parser client (new)Enable for Prometheus datasource",
```

Comment:
Nit. Might be wrong, but should `Enable for Prometheus datasource` be appended at the end?

Reply:
Yeah, that is not that much clear. I will update it and try to make it a bit more clear.

Reply:
Oh shit I just saw the words at the end :D 
Good catch!